### DISCUSSION TITLE
top header section under each of links, lists, tags

### DISCUSSION THREAD

**Discussion by habitats-tech** [Kovah/LinkAce#561] [Ideas & Feature Requests]
A top header section under each view (links, lists, tags), setup under settings with no text length restriction. Optionally ability to format the text using markdown notation. The header customisable public/private.

The header is useful to make notes when, for example, someone is building intelligence in the link/tag name. Sometimes the same tag name applies to two different domains (e.g. technology, science), but someone wants to differentiate between the two (e.g. tech-tag_name, science-tag_name).

**Comment by Kovah** [comment]
Could you elaborate on this a bit further? I am not quite sure if I understand what you mean with that header.

**Reply by habitats-tech** [reply]
Please see below my idea. Just the header with no bells & whistles should be a relatively painless task to implement.

![image](https://user-images.githubusercontent.com/87811684/198678485-d0483636-a353-457e-9b46-059ae26bd225.png)

Ability to show or not the header plus specify if public or private would be useful.

**Reply by Kovah** [reply]
Wouldn't a description for tags be sufficient for this, like there is for lists?

**Comment by habitats-tech** [comment]
The idea is to have a global header one for each tab Links, Lists, Tags. This header not particular to a specific list or tab or link. This header is one way of describing the structure of the tab (e.g. describe tab naming conventions). It would also be useful when in guest mode because it can provide a reminder why we chose to do things in a specific way, especially when long term is considered.
